scripts/g_r.py

```diff
@@ -0,0 +1,60 @@
+import datetime
```

Comment:
Your code should be self documenting when possible. If you can find a way to make it self documenting with less chars thats great, but at the moment this isn't something I would accept.